hd/lang/lex_utf8.txt

```diff
@@ -10,20 +10,6 @@ fr: Tapez à l'identique le nom de la généalogie que vous voulez utiliser :
 en: list of available genealogies (use gwsetup)
 fr: listes des généalogies disponibles (utilise gwsetup)
 
-    about (short date)
```

Comment:
As already said, this is clearly wrong in this PR. Which means it may brings conflicts in other PR. Only add the related modifications to one PR